src/app/component/header/header.component.html

```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
-      <img class="d-inline-block" src="assets/img/logo1.png" width="35" height="35" />
+      <img class="d-inline-block" src="assets/img/logo1.png" alt="Jyoti Techn" width="35" height="35" />
```

Comment:
Do not use h1 as className please remove it and give proper classname to it

Reply:
please move this inline css to external file

Reply:
yes I have  change inline CSS to external file